### PR TITLE
Update old samples to new agent using Room Reservations and match can…

### PR DESCRIPTION
**Pull request by nnegrey**
…onical samples in python

Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/resource.js [hunk 1]
```diff
@@ -924,59 +441,51 @@ function listContexts(projectId, sessionId) {
...
+        console.log(`Context name: ${context.name}`);
+        console.log(`Lifespan count: ${context.lifespanCount}`);
+        console.log('Fields:')
+        if (context.parameters != null) {
+          context.parameters.fields.forEach(field, value => {
+            console.log(`\t${field}: ${value}`);
+          });
+        }
+      });
       return responses[0];
```
Comment (happyhuman) [inline samples/resource.js]: Return may not be needed.

**Comment by JustinBeckwith** [comment]
@nnegrey I'd love to get these samples tests fixed so we can ship a new build :)

File: samples/resource.js [hunk 2]
```diff
@@ -116,60 +72,46 @@ function listEntityTypes(projectId) {
   return entityTypesClient
     .listEntityTypes(request)
     .then(responses => {
+      responses[0].forEach(entityType => {
+        console.log(`Entity type name: ${entityType.name}`);
+        console.log(`Entity type display name: ${entityType.displayName}`);
+        console.log(`Number of entities: ${entityType.entities.length}\n`);
+      })
       return responses[0];
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply samples/resource.js]: Not needed, but something we're slowly moving toward for a list method should return results.

**Comment by nnegrey** [comment]
@JustinBeckwith Fixed

**Pull request merged**